### PR TITLE
[DO-1858]Added command to save luxury cars and method is_vehicle_luxury

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -1415,8 +1416,73 @@ def save_securities(self, securities_data, declaration):
             self.save_right(securities, data)
 
     # TODO: implement
-    def is_vehicle_luxury(self, brand, model, year):
-        pass
+    def is_vehicle_luxury(self, brand, model, year, valuation):
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:1419]: Let's pass to the function the hole Vehicle object with type=Vehicle.Car and validate this

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -1415,8 +1416,73 @@ def save_securities(self, securities_data, declaration):
...
+        normalized_model = {
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:1420]: I think it`s better first to check is `valuation >= 800000`

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -1438,17 +1504,20 @@ def save_vehicle(self, vehicles_data, declaration):
         for data in vehicles_data:
             vehicle_type = types.get(data.get('objectType'))
             additional_info = data.get('otherObjectType', '')
-            brand = data.get('brand')
-            model = data.get('model')
-            year = data.get('graduationYear')
+            brand = data.get('brand') if data.get('brand') not in self.NO_DATA else ''
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:1507]: Are you sure that is a good idea? Bad names are better than no names

File: business_register/converter/declaration.py [hunk 4]
```diff
@@ -1438,17 +1504,20 @@ def save_vehicle(self, vehicles_data, declaration):
...
+            model = data.get('model') if data.get('model') not in self.NO_DATA else ''
+            year = data.get('graduationYear') if data.get('graduationYear') not in self.NO_DATA else None
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:1509]: same here

Comment (IegorSoboliev) [reply business_register/converter/declaration.py:1509]: and here

File: business_register/converter/declaration.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/declaration.py:1419]: Then we can use this function not only here

**Review by IegorSoboliev — CHANGES_REQUESTED** [review]

File: business_register/converter/declaration.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

File: business_register/converter/declaration.py (hunk 2 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

File: business_register/converter/declaration.py (hunk 3 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

File: business_register/converter/declaration.py (hunk 4 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

**Review by Andrii17s — APPROVED** [review]

File: business_register/pep_scoring/rules.py [hunk 5]
```diff
@@ -840,3 +840,31 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+class IsLuxuryCar(BaseScoringRule):
+    """
+    Rule 18 - PEP18
+    weight - 0.4, 0.1
+    Declared ownership and/or right of use of a business class car, or car with a price exceeding 800 000 UAH
+    or brand vehicle, which is considered to be a luxury car
+    """
+    rule_id = ScoringRuleEnum.PEP18
+    message_en = 'amount of luxury cars - {amount_luxury_cars}'
+    message_uk = 'кількість люксових авто - {amount_luxury_cars}'
```
Comment (IegorSoboliev) [inline business_register/pep_scoring/rules.py:855]: I would recommend a next text: 
`У декларації зазначено {amount_luxury_cars} авто, ціна яких більше 800 тисяч гривень або які входять да затвердженого урядом...(тут ти краще знаєш)`

**Review by IegorSoboliev — APPROVED** [review]

File: business_register/pep_scoring/rules.py [hunk 6]
```diff
@@ -840,3 +840,31 @@ def calculate_weight(self) -> Tuple[Union[int, float], dict]:
...
+    class DataSerializer(serializers.Serializer):
+        amount_luxury_cars = serializers.IntegerField(min_value=0, required=True)
+
+    def calculate_weight(self) -> Tuple[Union[int, float], dict]:
+        amount_luxury_cars = Vehicle.objects.filter(
+            declaration=self.declaration.id,
+            is_luxury=True,
+        ).all().count()
+        if amount_luxury_cars:
+            weight = 0.4 + (amount_luxury_cars - 1) * 0.1
```
Comment (IegorSoboliev) [inline business_register/pep_scoring/rules.py:866]: Is that approved by ANTAC)?

**Review by roman-tiukh — APPROVED** [review]

**Review by Andrii17s — APPROVED** [review]

**Review by IegorSoboliev — APPROVED** [review]

**Pull request merged**